src/app/routes/onDemandRadio/getInitialData/addIdsToBlocks.js

```diff
@@ -0,0 +1,19 @@
+import pipe from 'ramda/src/pipe';
+import path from 'ramda/src/path';
+import map from 'ramda/src/map';
+import mergeDeepLeft from 'ramda/src/mergeDeepLeft';
+import uuid from 'uuid';
+
+const addIdToBlock = block => ({ ...block, uuid: uuid() });
+
+const getBlocks = path(['content', 'blocks']);
+
+const mapBlocks = pipe(getBlocks, map(addIdToBlock));
+
+export default jsonRaw =>
+  mergeDeepLeft(
+    {
+      content: { blocks: mapBlocks(jsonRaw) },
+    },
+    jsonRaw,
+  );
```

Comment:
This appears to be a duplicate of https://github.com/bbc/simorgh/blob/latest/src/app/routes/radio/getInitialData/addIdsToBlocks.js - is it possible to try and reuse this somehow?

Reply:
You could pop it in here `src/app/routes/utils/sharedDataTransformers` and call it `addIdsToRadioBlocks` so both live radio and ondemand routes can use it

Reply:
Not sure if this is in scope for this PR as it's just supposed to be a blank page,  but I'm not really sure we need to be using the `Blocks` renderer for this page at all

It seems to be the case that ARES payloads for these pages only ever have one block - the block that  contains the episode info

We don't have to be iterating over blocks if there's only ever one block, and if we're not iterating over blocks, we don't need them to be adding IDs to the blocks

Reply:
Removed it